modules/weblog/README.md

```diff
@@ -154,14 +154,14 @@ In addition to that weblog understands [user defined fields](#custom-fields-feat
 Notes:
 
 -   Apache `%h` logs the IP address if [HostnameLookups](https://httpd.apache.org/docs/2.4/mod/core.html#hostnamelookups) is Off.
-    Weblog counts hostname as IPv4 address. We recommend either to disable HostnameLookups or use `%a` instead of `%h`. 
+    Weblog counts hostname as IPv4 address. We recommend either to disable HostnameLookups or use `%a` instead of `%h`.
```

Comment:
It's in a few other places in this doc, so we don't have to change them all now, but I think we should avoid calling it **Weblog**. I always read it first as "we blog." I think we should stick with these usages:

- The web log collector...
- Netdata's web log collector...
- The `web_log` collector...

```suggestion
    The web log collector counts hostnames as IPv4 addresses. We recommend either to disable HostnameLookups or use `%a` instead of `%h`.
```